.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 
 <!-- [x] instead of [ ] checks the task -->
 
-- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/main/CONTRIBUTING.md).
+- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/main/docs/developer/introduction.md).
```

Comment:
Thanks for the PR, we did indeed move the contributing guide so the link is wrong, it should now be https://github.com/openkfw/TruBudget/blob/main/.github/CONTRIBUTING.md